profiles.json

```diff
@@ -7,5 +7,8 @@
   },
    "LiveRaveN": {
     "avatar": "https://s3.amazonaws.com/keybase_processed_uploads/d33c27b78d11e9bd65af793637e31a05_360_360.jpg"
+  },
+   "gruberx_ursus": {
+    "avatar": "https://avatars.githubusercontent.com/u/35637653?s=360&v=4"
```

Comment:
This is okay! Just wanna give a heads up that the profile img will probably not show yet, since this domain will get blocked on my site. I will fix this when I push my next release. If you prefer to already see your avatar, then use an image coming from a keybase.io profile (they use the amazonaws domain).

Reply:
I'm fine with it. Thanks.